ZenPacks/zenoss/Import4/rrd2tsdb.py

```diff
@@ -158,7 +160,7 @@ def _process_v(self, tag_path, content):
         else:
             print '{} {:d} {:.10e} device={} key={} zenoss_tenant_id={}'.format(
                 self.metric, self.last_timestamp, _value,
-                self.device, self.key.replace(' ', '-'), self.dmd_uuid)
+                self.device, self.key.translate(self.tbl), self.dmd_uuid)
```

Comment:
Are you wanting to white list certain characters or black list certain characters? (ie - do you know exactly which characters are not acceptable and need to be translated? or do you know exactly which characters are acceptable, and you want to translate everything else?)